source/views/streaming/streams/list.tsx

```diff
@@ -28,6 +29,24 @@ const groupStreams = (entries: StreamType[]) => {
 	return toPairs(grouped).map(([title, data]) => ({title, data}))
 }
 
+const getEnabledCategories = (filters: ListType[]) => {
+	return filters.flatMap((filter: ListType) =>
+		filter.spec.selected.flatMap((spec) => spec.title),
+	)
+}
+
+const filterStreams = (streams: StreamType[], filters: ListType[]) => {
+	return streams.filter((stream) => {
+		let enabledCategories = getEnabledCategories(filters)
+
+		if (enabledCategories.length === 0) {
+			return stream
+		}
+
+		return enabledCategories.includes(stream.category)
+	})
+}
```

Comment:
```suggestion
const filterStreams = (streams: StreamType[], filters: ListType[]) => {
	let enabledCategories = getEnabledCategories(filters)

	if (enabledCategories.length === 0) {
		return streams
	}

	return streams.filter((stream) => enabledCategories.includes(stream.category))
}
```

Reply:
That does look nicer. Let's go with yours.